src/components/students-in-categories/StudentsInCategories.tsx

```diff
@@ -0,0 +1,98 @@
+import { Box, Button, MenuItem, Select, Typography } from '@mui/material'
+import { styles } from './StudentsInCategories.styles'
+import SchoolIcon from '@mui/icons-material/School'
+import EventIcon from '@mui/icons-material/Event'
+import StudentsInCatgoriesChart from './StudentsInCategoriesChart'
+import { useTranslation } from 'react-i18next'
+
+const currencies = [
+  {
+    value: 'USD',
+    label: '$'
+  },
+  {
+    value: 'EUR',
+    label: '€'
+  },
+  {
+    value: 'BTC',
+    label: '฿'
+  },
+  {
+    value: 'JPY',
+    label: '¥'
+  }
+]
+
+const years = [
```

Comment:
probably its better to use Date.getFullYear() and set -2 years or take date since user registered